frameworks/person-escort-record/README.md

```diff
@@ -76,10 +76,15 @@ Question files should live in [`questions/`](./questions) and be stored in a fla
 
 ### Question options
 
-- `type` **(required)** - the type of question input. Current supported types: `radio`, `checkbox`, `text`, `textarea`
+- `type` **(required)** - the type of question input. Current supported types: `radio`, `checkbox`, `text`, `textarea`, `multiple`
```

Comment:
I know we've been struggling a bit with the name of this. What about `add_multiple` - similar to the [Home Office pattern](https://home-office-digital-patterns.herokuapp.com/patterns/add-multiple-things)? Or something along the lines of _repeating_? `repeating_questions`? Not sure though...

I think `multiple` on its own could be a little ambiguous. Could mean multiple responses...multiple options...

Reply:
`question_loop`?